dev-python/sortedcontainers/sortedcontainers-2.1.0.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python2_7 python3_{6,7,8} )
```

Comment:
`sortedcontainers-1.5.7.ebuild` has Python 3.5 support enabled and upstream hasn't dropped support for 3.5. according to `setup.py` metadata. Why has 3.5 been dropped during the bump?

Reply:
@flappyports told me it could be dropped here: https://github.com/gentoo/gentoo/pull/13988

>py3_5 can be dropped as it will be dropped from Gentoo soon.

Reply:
Dropping Python 3.5 will need touching many ebuilds at once: a mass-removal. Before that happens, I don't see how thinning support out at some place helps the big picture unless python support 3.5 is known broken in that package. Am I missing something? @flappyports what do you think?